.eslint-dictionary.json

```diff
@@ -1,5 +1,14 @@
 [
+  "Cipheriv",
```

Comment:
cipher iv stands for cipher initialization vector and it's a technical term in cryptography. This is here because we have a [dependency on a cryptographic library with functions that have Cipheriv](https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-cli/src/commands/helpers/encryption-helpers.ts) in the name. That is why this should be here.

Reply:
discussed offline and agreed to have followup task/PR to cleanup the dictionary file from entries like `abcdefghijklmnopqrstuvwxyz`